src/sentry/demo/settings.py

```diff
@@ -0,0 +1,18 @@
+from datetime import timedelta
+from django.conf import settings
```

Comment:
You don't need to import `settings` here. You're already in the settings file, you can just access any attributes directly (they're imported via the `import *`)